src/search/components/SearchResults.js

```diff
@@ -7,16 +7,42 @@ export default class SearchResults extends React.Component {
   }
 
   static defaultProps = {
-    query: '',
+    query: null,
     results: [],
   }
 
   render() {
     const { query, results } = this.props;
+    const searchResultsClass = 'search-results';
+
+    let searchResults;
+
+    if (query && results.length > 0) {
+      searchResults = (
+        <div className={searchResultsClass}>
```

Comment:
It might be nicer to pull the `div` out of these blocks and do:

```js
if (query && results.length > 0) {
  message = `Your search for "${query}" returned ${results.length} results.`;
  searchResults = <ul ref="results">
    {results.map((result) => <li key={result.title}>{result.title}</li>)}
  </ul>;
} else if (query && results.length === 0) {
  message = `No results were found for "${query}".`;
} else { ... }

return <div ref="container" className="search-results">
  <p ref="message">{message}</p>
  {searchResults}
</div>;